docs/02. Quick Start.md

```diff
@@ -11,21 +11,21 @@ the README file.
 
 ## Specifying an identity provider
 
-By default, ZfcRbac internally uses the `Zend\Authentication\AuthenticationService` service key to retrieve the user (logged or
+By default, ZfcRbac internally uses the `Laminas\Authentication\AuthenticationService` service key to retrieve the user (logged or
 not). Therefore, you must implement and register this service in your application by adding these lines in your `module.config.php` file:
 
 ```php
 return [
     'service_manager' => [
         'factories' => [
-	        'Zend\Authentication\AuthenticationService' => function($sm) {
+	        'Laminas\Authentication\AuthenticationService' => function($sm) {
 	            // Create your authentication service!
 	        }
 	    ]
     ]
 ];
 ```
-The identity given by `Zend\Authentication\AuthenticationService` must implement `ZfcRbac\Identity\IdentityInterface`. Note that the default identity provided with ZF2 does not implement this interface, neither does the ZfcUser suite.
+The identity given by `Laminas\Authentication\AuthenticationService` must implement `ZfcRbac\Identity\IdentityInterface`. Note that the default identity provided with ZF2 does not implement this interface, neither does the ZfcUser suite.
```

Comment:
Might want to do a search for the abbreviation ZF2 and replace that with something else -- I see it here, and I imagine it is found elsewhere too.